student.rb

```diff
@@ -13,5 +13,5 @@ def play_hooky
   end
 end
 
-st = Student.new(10, 'Aku')
-p st.play_hooky
+# st = Student.new(10, 'Aku')
+# p st.play_hooky
```

Comment:
- [ ] Here as well. I would recommend removing the commented line to make the codebase cleaner.